sentry/src/main/java/io/sentry/ShutdownHookIntegration.java

```diff
@@ -41,7 +41,16 @@ public void register(final @NotNull IHub hub, final @NotNull SentryOptions optio
   @Override
   public void close() throws IOException {
     if (thread != null) {
-      runtime.removeShutdownHook(thread);
+      try {
+        runtime.removeShutdownHook(thread);
+      } catch (IllegalStateException e) {
+        @Nullable final String message = e.getMessage();
+        if (message != null && message.equals("Shutdown in progress")) {
```

Comment:
maybe it makes sense to add a link where this message is coming from?
```suggestion
        // https://github.com/openjdk/jdk/blob/09b8a1959771213cb982d062f0a913285e4a0c6e/src/java.base/share/classes/java/lang/ApplicationShutdownHooks.java#L83
        if (message != null && message.equals("Shutdown in progress")) {
```